### ISSUE TITLE
Display tags & contact persons in alphabetical order in hearing editor

### ISSUE THREAD

**Issue by petterill** [City-of-Helsinki/kerrokantasi-ui#537]
Previously created tags & contact persons seem to be displayed in random order. Should be listed in alphabetical order.

**Comment by jonnetanninen** [comment]
Does this have backend support @Rikuoja ?

**Comment by Rikuoja** [comment]
Yeah the contact persons are now displayed in alphabetic order in the backend, so no UI issue there.

However, the labels are trickier because their name is in several languages; this would mean we would have to fetch the labels again when the language is changed?

**Comment by Rikuoja** [comment]
Will add sorting parameter `name` support to backend, we should just use that.

**Comment by jussih** [comment]
Backend implementation has been completed in issues:
https://github.com/City-of-Helsinki/kerrokantasi/issues/267
https://github.com/City-of-Helsinki/kerrokantasi/issues/275
